Remora.Discord.Commands/Extensions/CommandTreeExtensions.cs

```diff
@@ -537,6 +558,12 @@ private static Result<IReadOnlyList<IApplicationCommandOption>> TryCreateCommand
         ILocalizationProvider localizationProvider
     )
     {
+        // Parameters options are only supported by slash commands
+        if (command.GetCommandType() is not ApplicationCommandType.ChatInput)
```

Comment:
What's with this?

Reply:
It's probably to try to guard against trying to parse any other parameters here as they are not supported for User/Message context commands?

Reply:
I just moved the check in the method that is calling `TryCreateCommandParameterOptions`. Should be more cleaner.

As AraHaan just said the check is required so that additional parameters are not mapped to options as those are not supported by User/Message application commands.